doc/resources/mi-cli-command-tree.mmd

```diff
@@ -0,0 +1,26 @@
+graph LR
```

Comment:
Can we add a comment explaining how to generate the image from this file?

Reply:
Fixed in fd3ef83